lib/ButtonNew/ButtonTertiary/ButtonTertiary.js

```diff
@@ -5,17 +5,17 @@ import { ButtonBase } from '../ButtonBase';
 import { propTypes, defaultProps, sizes } from '../common';
 
 function ButtonTertiary({ children, className, disabled, active, ...rest }) {
-  const classes = cx('bg-white focus:shadow-blue-focus-sm', className, {
-    'text-neutral-primary': disabled,
-    'text-blue-primary': !disabled,
-    'hover:bg-blue-95 active:bg-blue-90': !disabled && !active,
-    'bg-blue-90': active
+  const classes = cx('bg-white focus:shadow-neutral-focus-sm', className, {
```

Comment:
I'm refactoring (in a separate PR) these double border/box shadows to use a single border and a box shadow - just mentioning this feedback as it's related work. I'll rebase on this commit though 👍

Reply:
Great - lets keep this in mind!